interface-elements-for-desktop/articles/rich-text-editor/track-changes/accept-and-reject-tracked-changes.md

```diff
@@ -0,0 +1,53 @@
+# Accept and Reject Changes
+
+## Accept Changes
+
+### Accept A Specific Change
+
+To accept a single change, select the change you want to accept and click **Accept** button in the **Changes** ribbon group on the **Review** tab. The change will be accepted, and the cursor will be moved to the next change.
+
+![IMAGE](../../../images/rich-text-editor-track-changes-accept.png)
+
+To accept the change and keep the cursor at the same position, open the **Accept** drop down menu and select **Accept This Change**.
+
+![IMAGE](../../../images/rich-text-editor-track-changes-accept-this-change.png)
+
+You can also accept a change from the context menu.
+
+![IMAGE](../../../images/rich-text-editor-track-changes-accept-context-menu.png)
+
+### Accept All Changes
+
+You can accept all changes at once. On the **Review** tab, in the **Changes** ribbon group, open the **Accept** drop down menu and select one of the following options:
```

Comment:
```suggestion
All the changes can be accepted simultaneously. In the **Changes** ribbon group on the **Review** tab, open the **Accept** drop-down menu and select one of the following options:
```

Reply:
Why are all these sentences backwards? and why do you write 'drop down' when it should be 'drop-down' since it is an adjective?